coreneuron/io/nrn_setup.hpp

```diff
@@ -118,7 +118,7 @@ inline void* phase_wrapper_w(NrnThread* nt, UserParams& userParams, bool in_memo
 
             // Avoid trying to open the gid_gap.dat file if it doesn't exist when there are no
             // gap junctions in this gid
-            if (P == gap && !userParams.file_reader[i].file_exist(fname)) {
+            if (P == gap && !FileHandler::file_exist(fname)) {
                 userParams.file_reader[i].close();
             } else {
```

Comment:
I thought I understood this, but in fact no. Can someone explain to me?

Reply:
Added some explanation now.